### ISSUE TITLE
Can't change the text of the Buttons, and Titles.

### ISSUE THREAD

**Issue by neumannpatrik** [elementor/elementor#16782]
## Prerequisites

- [ x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x ] The issue still exists against the latest stable version of Elementor.

## Description

In the new Cart widget, I can't change the text of the Buttons, and Titles.

## Steps to reproduce

Add the Cart widget.
![image](https://user-images.githubusercontent.com/59146770/139757732-a2e4c8c5-d85b-43d5-82d7-5eafa3749b25.png)

Write the new text.
![image](https://user-images.githubusercontent.com/59146770/139757808-38b7ebbb-fead-4b44-a8b4-7c102ec20028.png)

In the editor it is updated:
![image](https://user-images.githubusercontent.com/59146770/139757883-c67bac36-e7ed-49b2-adf6-f6547bfec56e.png)

But in the live page, it is not:
![image](https://user-images.githubusercontent.com/59146770/139757949-1f8ff335-9c7e-4545-89f9-05792609a318.png)

## Isolating the problem

- [x ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x ] This bug happens with a default WordPress theme active.
- [x ] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>

== Server Environment ==
	Operating System: WINNT
	Software: Apache/2.4.46 (Win64) OpenSSL/1.1.1h PHP/8.0.2
	MySQL version: mariadb.org binary distribution v10.4.17
	PHP Version: 8.0.2
	PHP Max Input Vars: 1000
	PHP Max Post Size: 40M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.8.1
	Site URL: http://localhost/femzol.hu
	Home URL: http://localhost/femzol.hu
	WP Multisite: No
	Max Upload Size: 40 MB
	Memory limit: 512M
	Permalink Structure: /%year%/%monthnum%/%day%/%postname%/
	Language: hu
	Timezone: 0
	Admin Email: patrikneumann3@gmail.com
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.4.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: hu_HU
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.54 Safari/537.36

== Active Plugins ==
	Cookie Notice &amp; Compliance for GDPR / CCPA
		Version: 2.1.5
		Author: Hu-manity.co

	Dashboard Welcome for Elementor
		Version: 1.0.5
		Author: IdeaBox Creations

	Elementor
		Version: 3.4.7
		Author: Elementor.com

	Elementor Pro
		Version: 3.5.0-beta4
		Author: Elementor.com

	Google Analytics for WordPress by MonsterInsights
		Version: 8.1.0
		Author: MonsterInsights

	Themify - WooCommerce Product Filter
		Version: 1.3.5
		Author: Themify

	TranslatePress - Multilingual
		Version: 2.1.5
		Author: Cozmoslabs, Razvan Mocanu, Madalin Ungureanu, Cristophor Hurduban

	White Label CMS
		Version: 2.2.4
		Author: www.videousermanuals.com

	WooCommerce
		Version: 5.8.0
		Author: Automattic

== Elementor - Compatibility Tag ==
	
	Dashboard Welcome for Elementor: Compatibility not specified
	Elementor Pro: Összeillő

== Elementor Pro - Compatibility Tag ==
	

== Elements Usage ==
	
	footer : 3
		heading : 9
		icon-list : 6
		image : 3
	header : 10
		button : 2
		heading : 6
		icon-list : 8
		image : 4
		nav-menu : 4
		shortcode : 3
		woocommerce-menu-cart : 2
	page : 1
		button : 4
		icon-box : 4
		image : 1
	product : 1
		heading : 1
		woocommerce-breadcrumb : 1
		woocommerce-product-add-to-cart : 1
		woocommerce-product-data-tabs : 1
		woocommerce-product-images : 1
		woocommerce-product-price : 1
		woocommerce-product-rating : 1
		woocommerce-product-related : 1
		woocommerce-product-short-description : 1
		woocommerce-product-title : 2
	section : 3
		divider : 7
		heading : 12
		icon : 1
		image : 6
		text-editor : 8
		woocommerce-menu-cart : 1
	wp-page : 26
		button : 2
		divider : 14
		form : 1
		gallery : 1
		google_maps : 1
		heading : 38
		icon : 9
		icon-list : 2
		image : 22
		nav-menu : 1
		text-editor : 37
		woocommerce-cart : 1
		woocommerce-checkout-page : 1
		woocommerce-my-account : 1
	product-archive : 1
		shortcode : 1
		woocommerce-products : 1

== Log ==
	
PHP: showing 4 of 4PHP: 2021-10-24 14:41:18 [error X 3][P:\Programs\XAMPP\htdocs\femzol.hu\wp-content\plugins\elementor-pro\modules\theme-builder\classes\conditions-manager.php::285] Uncaught Error: Call to a member function update_meta() on bool in P:\Programs\XAMPP\htdocs\femzol.hu\wp-content\plugins\elementor-pro\modules\theme-builder\classes\conditions-manager.php:285
Stack trace:
#0 P:\Programs\XAMPP\htdocs\femzol.hu\wp-content\plugins\elementor-pro\modules\theme-builder\classes\conditions-manager.php(166): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager-&gt;save_conditions(NULL, Array)
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager-&gt;ajax_save_theme_template_conditions(Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#2 P:\Programs\XAMPP\htdocs\femzol.hu\wp-content\plugins\elementor\core\common\modules\ajax\module.php(171): call_user_func(Array, Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#3 P:\Programs\XAMPP\htdocs\femzol.hu\wp-includes\class-wp-hook.php(303): Elementor\Core\Common\Modules\Ajax\Module-&gt;handle_ajax_request(&#039;&#039;)
#4 P:\Programs\XAMPP\htdocs\femzol.hu\wp-includes\class-wp-hook.php(327): WP_Hook-&gt;apply_filters(&#039;&#039;, Array)
#5 P:\Programs\XAMPP\htdocs\femzol.hu\wp-includes\plugin.php(470): WP_Hook-&gt;do_action(Array)
#6 P:\Programs\XAMPP\htdocs\femzol.hu\wp-admin\admin-ajax.php(187): do_action(&#039;wp_ajax_element...&#039;)
#7 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-10-26 16:28:50 [warning X 3][P:\Programs\XAMPP\htdocs\femzol.hu\wp-content\plugins\elementor\includes\managers\image.php::87] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-10-28 18:04:19 [warning X 7][P:\Programs\XAMPP\htdocs\femzol.hu\wp-content\plugins\elementor\core\common\modules\ajax\module.php::171] Undefined array key &quot;data&quot; [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-10-31 15:27:24 [error X 3][P:\Programs\XAMPP\htdocs\femzol.hu\wp-content\plugins\elementor-master\elementor.php::66] Cannot redeclare elementor_load_plugin_textdomain() (previously declared in P:\Programs\XAMPP\htdocs\femzol.hu\wp-content\plugins\elementor\elementor.php:67) [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

Log: showing 19 of 192021-10-31 14:38:27 [info] elementor::elementor_updater Started 
2021-10-31 14:38:27 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.4.6',
  'to' => '3.4.7',
)]
2021-10-31 14:38:28 [info] Elementor/Upgrades - _on_each_version Start  
2021-10-31 14:38:31 [info] Elementor/Upgrades - _on_each_version Finished 
2021-10-31 14:38:31 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.4.6',
  'to' => '3.4.7',
)]
2021-10-31 19:03:27 [info] elementor-pro::elementor_pro_updater Started 
2021-10-31 19:03:27 [info] Elementor Pro/Upgrades - _on_each_version Start  
2021-10-31 19:03:27 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.4.2',
  'to' => '3.5.0-beta3',
)]
2021-10-31 19:03:27 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2021-10-31 19:03:27 [info] Elementor Pro/Upgrades - _v_3_5_0_price_list Start  
2021-10-31 19:03:28 [info] Elementor Pro/Upgrades - _v_3_5_0_price_list Finished 
2021-10-31 19:03:29 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.4.2',
  'to' => '3.5.0-beta3',
)]
2021-11-01 20:12:48 [info] elementor-pro::elementor_pro_updater Started 
2021-11-01 20:12:48 [info] Elementor Pro/Upgrades - _on_each_version Start  
2021-11-01 20:12:48 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2021-11-01 20:12:48 [info] Elementor Pro/Upgrades - _v_3_5_0_price_list Start  
2021-11-01 20:12:48 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.5.0-beta3',
  'to' => '3.5.0-beta4',
)]
2021-11-01 20:12:49 [info] Elementor Pro/Upgrades - _v_3_5_0_price_list Finished 
2021-11-01 20:12:49 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.5.0-beta3',
  'to' => '3.5.0-beta4',
)]

**Comment by avivu** [comment]
Hello @neumannpatrik,

Please try:

- Deactivate all your plugins besides Elementor, Elementor Pro and WooCommerce. If this solves the problem, gradually activate your plugins one by one until you spot the problematic plugin.

And see if the issue persists.

**Comment by gilesholder** [comment]
Have you tried clearing your cache? Works for me on testing.

**Comment by neumannpatrik** [comment]
Hi, the problem was TranslatePress plugin.

Thank you for your help!
